test/specs/login_page.spec.js

```diff
@@ -0,0 +1,87 @@
+const LoginPage = require("../pages/LoginPage");
+const login = require("../utils/login.json")
+
+describe("Sauce Tests", () => {
+
+  it("Login Test Standard", async () => {
+
+    browser.url(login.url);
+
+    await LoginPage.loginCSS(login.username[0], login.password);
+
+    await LoginPage.checkTitle(login.title);
```

Comment:
Use soft assertion

Reply:
Use it in the 'Login Test Empty', add an array with two parameters